apps/deploy-web/package.json

```
@@ -49,6 +49,7 @@
     "@sentry/nextjs": "^7.80.0",
     "@stripe/react-stripe-js": "^1.12.0",
     "@stripe/stripe-js": "^1.41.0",
+    "@tanstack/react-query": "^5.53.2",
```

it seems like this version is the latest. Any reason why it's not used?
```suggestion
    "@tanstack/react-query": "^5.54.1",
```